Lib9c/Renderer/ActionRenderer.cs

```diff
@@ -42,6 +42,7 @@ IAccountStateDelta nextStates
                 BlockIndex = context.BlockIndex,
                 OutputStates = nextStates,
                 PreviousStates = context.PreviousStates,
+                RandomSeed = context.Random.Seed
```

Comment:
I get an error in 9c.
`'IRandom' does not contain a definition for 'Seed' and no accessible extension method 'Seed' accepting a first argument of type 'IRandom' could be found (are you missing a using directive or an assembly reference?)`

Reply:
We need to update and merge [9c side](https://github.com/planetarium/NineChronicles/pull/627/) too after merging this PR.